### PR TITLE
Oh, uhh, that isn't smoke! It's steam. Hamburgers have a 1% chance to spawn as steamed hams.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: Plain hamburgers may now spawn as steamed hams with a very low chance.
/:cl:

Steam from the steamed clams we're having! Mmm, steamed clams.

Plain hamburgers now have a 1% chance to spawn as steamed hams, which taste like fast food hamburger and have a unique description.

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_burgers.dm [hunk 1]
```diff
@@ -13,6 +13,19 @@
 	bonus_reagents = list("vitamin" = 1)
 	foodtype = GRAIN | MEAT
 
+/obj/item/reagent_containers/food/snacks/burger/plain/Initialize()
+	. = ..()
+	if(prob(1))
+		new/obj/effect/particle_effect/smoke(get_turf(src))
+		playsound(src, 'sound/effects/smoke.ogg', 50, TRUE)
+		visible_message("<span class='warning'>Oh, ye gods! [src] is ruined! But what if...?</span>")
+		name = "steamed ham"
```
Comment (Hatterhat) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]: Steamed hams? Localized entirely within my research station?

Comment (DaedalusGame) [reply code/modules/food_and_drinks/food/snacks_burgers.dm]: May I see it?

Comment (LetterN) [reply code/modules/food_and_drinks/food/snacks_burgers.dm]: No

**Comment by uraniummeltdown** [comment]
SNEED

**Comment by arsserpentarium** [comment]
Can't fully understand idea of that PR.

**Comment by 81Denton** [comment]
@arsserpentarium https://www.youtube.com/watch?v=Y4lnZr022M8

**Comment by Ordonis** [comment]
Please tell me that you're adding in an aurora borealis effect for when the Steamed Hams spawns. Also add in a Steamed Clams (Mmm, steamed clams) recipe.

**Comment by Xhuis** [comment]
A "space aurora borealis" effect might actually be really cool for a rare weather effect on the station that makes space look really pretty for a few minutes. I'm not sure how to do that, though.

**Comment by PKPenguin321** [comment]
Just change the light color of starlights to fluctuate through the lighter blues and violets

**Comment by Xhuis** [comment]
I dunno if that's possible, but if it is, I'd definitely do it.

**Pull request merged**